identity/mock-iot-server/src/hub.rs

```diff
@@ -3,7 +3,7 @@
 use crate::server::Response;
 
 fn list_modules(
-    modules: &std::collections::BTreeSet<aziot_identity_common::hub::Module>,
+    modules: &std::collections::HashSet<aziot_identity_common::hub::Module>,
```

Comment:
This is for correctness or optimization?

Reply:
Optimization. Ordering doesn't matter for this, so it's fine to use HashSet.